modules/modifiers_linear_gradient/src/Plugin/modifiers/LinearGradientModifier.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Drupal\modifiers_linear_gradient\Plugin\modifiers;
+
+use Drupal\modifiers\Modification;
+use Drupal\modifiers\ModifierPluginBase;
+
+/**
+ * Provides a Modifier to set the linear gradient on an element using custom colors.
```

Comment:
This line exceeds 80 characters, please shorten that - phpcs need short description on single line. I would stay with "Provides a Modifier to set the linear gradient on an element."